soroban-sdk/src/env.rs

```diff
@@ -244,30 +250,9 @@ impl Env {
     pub fn log_value<V: IntoVal<Env, RawVal>>(&self, v: V) {
         internal::Env::log_value(self, v.into_val(self));
     }
-
-    #[doc(hidden)]
-    pub fn get_ledger_version(&self) -> u32 {
-        internal::Env::get_ledger_version(self).try_into().unwrap()
-    }
-
-    #[doc(hidden)]
-    pub fn get_ledger_sequence(&self) -> u32 {
-        internal::Env::get_ledger_sequence(self).try_into().unwrap()
-    }
-
-    #[doc(hidden)]
-    pub fn get_ledger_timestamp(&self) -> u64 {
-        let obj = internal::Env::get_ledger_timestamp(self);
-        internal::Env::obj_to_u64(self, obj)
-    }
-
-    #[doc(hidden)]
-    pub fn get_ledger_network_id(&self) -> Bytes {
-        let bin_obj = internal::Env::get_ledger_network_id(self);
-        bin_obj.in_env(self).try_into().unwrap()
-    }
 }
 
+use crate::ledger::Ledger;
```

Comment:
Could this be inserted at the top of the file with the other imports? The imports where it has been grouped are testutils imports.